path_keys_register_test.go

```diff
@@ -57,13 +56,12 @@ func TestPathKeysRegister_Write(t *testing.T) {
 			tc := tc
 
 			t.Run(tc.name, func(t *testing.T) {
-
 				key := path.Base(tc.cryptoKey)
 
 				b, storage := testBackend(t)
 				_, err := b.HandleRequest(context.Background(), &logical.Request{
 					Storage:   storage,
-					Operation: logical.CreateOperation,
+					Operation: logical.UpdateOperation,
```

Comment:
Why is this change required?

Reply:
> Why is this change required?

I believe that is related to: https://github.com/hashicorp/vault/pull/18492

Reply:
I think the failure may have been introduced in #20

Reply:
@benashz Yep, exactly. Without this change the `TestPathKeysRegister_Write` tests were failing with 

```console
path_keys_register_test.go:78: unsupported operation
```